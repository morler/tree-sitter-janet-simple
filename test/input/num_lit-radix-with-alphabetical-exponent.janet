11r1&a
