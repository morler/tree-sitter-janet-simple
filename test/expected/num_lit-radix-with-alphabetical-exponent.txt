(source [0, 0] - [1, 0]
  (num_lit [0, 0] - [0, 6]))
